### DISCUSSION TITLE
HTML Tags not working with trans

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [pimcore/pimcore#15344] [Q&A / Support]
Hi,

I have added an HTML tag in the Translations but it is not working as expected.
![image](https://github.com/pimcore/pimcore/assets/60934941/d6429bd1-b34b-4e7d-9071-b92892d74c1b)

**Comment by twin-elements** [comment]
Did you added `raw` filter in twig?
`{{ "my_translation" | trans | raw }}`
